src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -196,7 +205,7 @@ object WdlExpression {
       }
       case a:Ast if a.getName == "FunctionCall" => {
         val name = a.getAttribute("name").asInstanceOf[Terminal].getSourceString
-        val params = a.getAttribute("params").asInstanceOf[AstList].asScala.toVector.map {a => toString(a, highlighter)}
+        val params = a.getAttribute("params").astListAsVector map {a => toString(a, highlighter)}
```

Comment:
Replace the `a =>` with using underscore.

Also seeing this makes it clear to me that `toString` should be set up to be called as a method, e.g. `_.toString(highlighter)`